CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ The most recent changes are listed first.
 
 ### Added
 
+- Add block header to GetBlock and GetBlockRange output (issue 527)
```

Comment:
This should mention the .proto update (once it has been brought in via subtree merge and once we've sorted out the tagging on the zcash/lightwallet-protocol repo.)